schema/logs/sample/http_server-log.json

```diff
@@ -0,0 +1,22 @@
+{
+  "@timestamp": "2022-12-09T10:39:23.000Z",
```

Comment:
Is timestamp compulsory field? I see it in all the log examples.

Reply:
yes it is

Reply:
con you mention this somewhere in the doc, if not already.